src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/imports/avoidstaticimport/Example1.java

```diff
@@ -6,9 +6,13 @@
 </module>
 */
 
+package com.puppycrawl.tools.checkstyle.checks.imports.avoidstaticimport;
+
 // xdoc section -- start
 import static java.lang.Math.pow;          // violation
 import static java.lang.System.*;          // violation
 import java.io.File;                       // OK
 import java.util.*;                        // OK
+
+class Example1{}
 // xdoc section -- end
```

Comment:
Please move this comment to be right after last `import`. This will resolve changes in xdoc files.

CI highlights that changes in xdoc files should be part of commit.
After execution of `mvn test` there should be no changed files.

Reply:
done